gradle.properties

```diff
@@ -1,6 +1,6 @@
 libGroup=com.smartcar.sdk
 libName=java-sdk
-libVersion=1.0.5
+libVersion=1.0.6
```

Comment:
This should really be automated, but for now you need to manually run `gradle markdown` after bumping the version to ensure the markdown docs reflect the new value.

(This is especially important because any download links for the current version are also updated)